src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/LootBoxes/Scripts/Scene/LootBox.cs.meta

```diff
@@ -1,3 +0,0 @@
-﻿fileFormatVersion: 2
```

Comment:
The imported sample for LootBoxes new package wouldn't be in this directory, It would be `Assets/Samples/web3.unity SDK Lootboxes/2.5.0/...`, making samples importable and reimporting samples should fix this.

Reply:
Can we also duplicate modified sample files in the UnitySampleProject automatically on PR to main just like Web3.Unity's Samples? should just require a few added lines on the action yml

Reply:
That is out of the scope for this issue imo. Tomorrow I am going to properly organize all the packages and samples so that nothing breaks when you freshly import packages, and that will also include updating the action script to import all the samples from packages to our Sample project. (both web3 auth and lootboxes).

Reply:
As long as we create an issue for it, it's fine by me :)

Reply:
So 3 things to mention in the issue,
1. Import Lootboxes sample into `UnitySampleProject`
2.  `SampleInitializer.cs` to add any scenes that need to be added in `EditorBuildSettings`
3. Add a job for duplicating samples in Auto Samples Duplicate GitHub action.

We can tackle 2 & 3 together when the issue is moved out of the backlog.

Reply:
2. `SampleInitializer` can also be used for any other sample initializations we need to do. I can resolve this conversation once the issue is created.